src/types/Search.ts

```diff
@@ -15,3 +15,4 @@ export type FieldOptionsMap = { [key: string]: { partial: boolean; values: (stri
 
 /** Search request payload that requires a search node to be specified. */
 export type SearchRequestPayload = components['schemas']['SearchRequestPayload'] & { search: SearchNodePayload };
+export type OptionalSearchRequestPayload = components['schemas']['SearchRequestPayload'];
```

Comment:
It is optional in the API according to the commit message for the git blame on the previous line.